CRM/Utils/Mail/EmailProcessor.php

```diff
@@ -141,30 +136,21 @@ private static function _process($civiMail, $dao, $is_create_activities) {
         if ($usedfor == 1) {
           foreach ($mail->to as $address) {
             if (preg_match($regex, ($address->email ?? ''), $matches)) {
-              [$match, $action, $job, $queue, $hash] = $matches;
-              break;
-              // FIXME: the below elseifs should be dropped when we drop legacy support
-            }
-            elseif (preg_match($commonRegex, ($address->email ?? ''), $matches)) {
-              [$match, $action, $_, $job, $queue, $hash] = $matches;
-              break;
-            }
-            elseif (preg_match($subscrRegex, ($address->email ?? ''), $matches)) {
-              [$match, $action, $_, $job] = $matches;
+              [, $action, $job, $queue, $hash] = $matches;
```

Comment:
note `$match` is never used & now we have the option to not set in this construct